modules/retail/pages/retail-formulas.adoc

```diff
@@ -322,7 +322,8 @@ The VsFTPd formula is used to configure the FTP service on the branch server.
 . In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
 . Select the [systemitem]``Vsftpd`` formula, and click btn:[Save].
 . Navigate to the menu:Formulas[Vsftpd] tab, and set these parameters:
-* In the [guimenu]``Internal Network Address``, enter IP address of branch server (for example: [systemitem]``192.168.1.5``).
+* In the [guimenu]``FTP server directory`` field, enter [systemitem]``/srv/saltboot``.
+* In the [guimenu]``Internal Network Address`` field, enter the IP address of the branch server (for example: [systemitem]``192.168.1.5``).
```

Comment:
```suggestion
* In the [guimenu]``Internal Network Address`` field, enter the IP address of the branch server (for example: [literal]``192.168.1.5``).
```
Because it's a value, I'd go for ``literal``.